contracts/rounds/AdjustableRoundsManager.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.4.17;
+
+import "./RoundsManager.sol";
+
+
+contract AdjustableRoundsManager is RoundsManager {
```

Comment:
This contract is just used for integration tests and not for production deployment. It overrides `blockNum()` to return a state variable that can be modified by custom setters